genre.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'item'
+class Genre < Item
+  attr_accessor :id, :name, :items
+
+  def initialize(name)
+    super(publish_date, archived: false)
+    @id = Random.rand(1...1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.genre = self
+  end
+
+  def list_all_genres
+    puts 'No Genres added!' if @items.empty?
+    @items.each { |item| puts (item['genre']).to_s }
+    puts ''
+  end
```

Comment:
Kindly remove this method since it is not part of the requirement for this particular task.

Reply:
resolved

Reply:
Please remove the list_all_genres method, since it is not part of the current task requirements. 

`def list_all_genres
    puts 'No Genres added!' if @items.empty?
    @items.each { |item| puts (item['genre']).to_s }
    puts ''
  end`